outputs/opsgenie.go

```diff
@@ -0,0 +1,107 @@
+package outputs
+
+import (
+	"encoding/json"
+	"github.com/aquasecurity/postee/v2/formatting"
+	"github.com/aquasecurity/postee/v2/layout"
+	"log"
+
+	"github.com/opsgenie/opsgenie-go-sdk-v2/alert"
+	"github.com/opsgenie/opsgenie-go-sdk-v2/client"
+)
+
+const defaultPriority = alert.P3
+
+type OpsGenieOutput struct {
+	Name       string
+	User       string
+	APIKey     string
+	Responders []string
+	VisibleTo  []string
+}
+
+func (ops *OpsGenieOutput) GetName() string {
+	return ops.Name
+}
+
+func (ops *OpsGenieOutput) Init() error {
+	log.Printf("Starting OpsGenie output %q....", ops.Name)
+	return nil
+}
+
+func getUserResponders(users []string) []alert.Responder {
+	if len(users) == 0 {
+		return nil
+	}
+	responders := []alert.Responder{}
+	for _, user := range users {
+		responder := alert.Responder{Type: alert.UserResponder, Username: user}
+		responders = append(responders, responder)
+	}
+	return responders
+}
+
+func getString(i interface{}) string {
+	if i == nil {
+		return ""
+	}
+	return i.(string)
```

Comment:
ditto as other comment, maybe we should combine such checks as there's more than one place? what do you think?